src/lib/annotations/Annotator.js

```diff
@@ -236,10 +237,11 @@ class Annotator extends EventEmitter {
      * Sets the zoom scale.
      *
      * @param {number} scale - current zoom scale
-     * @return {void}
+     * @return {Annotator} Annotator instance returned for chaining
      */
     setScale(scale) {
         this.annotatedElement.setAttribute('data-scale', scale);
+        return this;
```

Comment:
Anytime you can chain you will have access to the instance. Its just for readability so you can chain calls in a clear readable fashion.

```
NumberArray.setFirstFiveElements(5)
                      .setMultiplier(3)
                      .transform()
                      .collect()
```
As opposed to
```
NumberArray.setFirstFiveElements(5)
NumberArray.setMultiplier(3)
NumberArray.transform()
NumberArray.collect()
```
Very minute semantic idea, although I understand it is probably not a good idea to throw it into the mix now. Essentially your functions will build properties of your instance and a few getter methods will actually display the transformed properties.